### DISCUSSION TITLE
Wand SW6 and SW7

### DISCUSSION THREAD

**Discussion by Beelzebot42** [gpstar81/GPStar-proton-pack#86] [General]
I am having an issue with the wand. SW6 (Mode switch) and SW7 (barrel switch). When activated they shut everything off, wand and pack. It seems to me that the way they are wired shorts the +/- directly to each other, which makes sense that would cause the pack to shut down. Wiring follows what is in the schematic, and I have confirmed I have the correct resistor in place. Any ideas as to what the issue might be? Otherwise things appear to be working. 

Should the resistor be between the (-) and the junction, rather than as shown where it is between the nano and the junction?

**Comment by Beelzebot42** [comment]
I moved the resistor between the signal wire and the (-) bus bar after researching more about pull-down resistors. It seems to have fixed the issue. I think the wiring schematic needs to be updated?

**Comment by gpstar81** [comment]
Ah yes I just noticed on the minimal schematic it's wrong. The resistor should be between ground and then connecting to the wire that runs from the switch to the nano pin.

**Reply by Yanks1023** [reply]
Has this been updated on the schematic yet? Im in the process of "building" the wand on the bench to test.

**Reply by gpstar81** [reply]
Yes it has.
